interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -446,7 +446,7 @@ function delete_me(delete_id) {
                 ?>
                     <div id="RXID_<?php echo attr($row['id']); ?>"
                          style="position:relative;text-align:center;margin: 10 auto;">
-                        <i class="float-right fa fa-close"
+                        <i class="float-right fas fa-times"
                            onclick="delete_me('<?php echo attr(addslashes($row['id'])); ?>');"
                            title="<?php echo xla('Remove this Prescription from the list of RXs dispensed'); ?>"></i>
                         <table style="margin:2px auto;width:490px;">
```

Comment:
Styling Error: Use table-responsive class and table class when using tables. Also try to remove the width attribute here by using either the grid classes or percent width as this implementation isn't responsive.